pages/_document.js

```diff
@@ -1,12 +1,37 @@
-import Document, { Html, Head, Main, NextScript } from 'next/document'
-// import { ServerStyleSheet } from 'styled-components'
+import Document, { Html, Head, Main, NextScript, DocumentContext } from 'next/document'
```

Comment:
remove DocumentContext as it is not used. Missing semicolons in a number of locations